homework/index.js

```diff
@@ -30,21 +30,53 @@
   }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    createAndAppend('p', ul, { text: `Repository:` });
+    createAndAppend('p', ul, { text: `Description:` });
+    createAndAppend('p', ul, { text: `Forks:` });
+    createAndAppend('p', ul, { text: `Updated:` });
+  }
```

Comment:
You are rendering the text labels and values in two separate, side-by-side divs. This works as if each row fits on one line. But is if they take more than one line, things get messy, as in the screenshot below, for a small screen device. Do you notice that the values for **Forks** and **Updated** for the **Git** repo are no longer aligned with their labels? Can you find another way of doing this that fixes this problem?

![image](https://user-images.githubusercontent.com/2788771/65252069-9ea70000-daf8-11e9-8e68-d465c39c0b75.png)

Reply:
Hi Jim,
Thank you for your feedback. 
I tried to fix problems that you text in your feedback. I changed some codes and now I can't find a solution. It is so complicated. I need more time to work and understand the logic of this code.

Reply:
Hi @ahmthsrvsmz, I think the complication lies in the HTML structure you have chosen. Trying this fix using JavaScript while keeping the underlying HTML is complicated if not impossible.

The best way to handle this is to plan the HTML you want to generate ahead of actually writing the corresponding JavaScript code.

The most straightforward way to render the repository details is to use an HTML `<table>`, like this:

```html
<ul>
  <li>
    <table>
      <tbody>
        <tr>
          <th class="label">Repository:</th>
          <td>
            <a href="https://github.com/HackYourFuture/alumni" target="_blank"
              >alumni</a
            >
          </td>
        </tr>
        <tr>
          <th class="label">Description:</th>
          <td>Alumni Guides, everything you need to land a job</td>
        </tr>
        <tr>
          <th class="label">Forks:</th>
          <td>10</td>
        </tr>
        <tr>
          <th class="label">Updated:</th>
          <td>6/21/2019, 1:38:21 PM</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
    <!-- etc-->
  </li>
</ul>
```

Once you have that plan, you can now write the JavaScript required to generate this HTML.

Does this help to get you on your way?

Reply:
I think this is a real hint. I will try again. Thank you @remarcmij